docs/source/usage/environmentvars.rst

```diff
@@ -40,6 +40,12 @@ done by setting the variable with no content::
     in other locations as well, amongst those are the users home directory,
     ``/usr/share/proj`` and the current folder.
 
+    You can also set the location of the resource files using 
+    :func:`proj_context_set_search_paths` in the `proj.h` API header. 
+    Starting with PROJ version 6.0.1, the :func:`proj_context_set_search_paths` will 
```

Comment:
This will be a 6.1.0 thing. A better way to to it is:

```
    ...
    in other locations as well, amongst those are the users home directory,
    ``/usr/share/proj`` and the current folder.	    ``/usr/share/proj`` and the current folder.

    .. versionchanged:: 6.1.0

        You can also set the location of ...
```

Reply:
Just updated it.